recipes-tests/pom.xml

```diff
@@ -410,7 +420,6 @@
                                     <groupId>jakarta.servlet</groupId>
                                     <artifactId>jakarta.servlet-api</artifactId>
                                     <version>${test.camel-quarkus-3-15.jakarta-servlet-api-version}</version>
-                                    <outputDirectory>${rewrite-tmp-classpath}</outputDirectory>
```

Comment:
Why this change?

Reply:
To be honest, I have absolutely no idea. I'll put it back where it belongs.